examples/basic/jsx-seconds-elapsed/src/main.js

```diff
@@ -5,7 +5,7 @@ import {timeDriver} from '@cycle/time';
 import {html} from 'snabbdom-jsx';
 
 function main(sources) {
-  const vdom$ = sources.Time.periodic(1000).map(i => i + 1).startWith(0)
+  const vdom$ = sources.Time.periodic(1000).startWith(0)
```

Comment:
The map is necessary, otherwise the counter has a duplicate `0` in the beginning.